data_prep/generate_climos.py

```diff
@@ -52,34 +37,20 @@ def iter_matching(dirpath, regexp):
             if regexp.match(abspath):
                 yield abspath
 
-def find_var_name(keys):
-    to_remove = [u'lat', u'lat_bnds', u'lon', u'lon_bnds', u'height', u'time', u'climatology_bounds']
-    varnames = [x for x in keys if x not in to_remove]
-    if len(varnames) == 1:
-        return varnames[0]
-    else:
-        raise
-
-def var_trans(variable):
-    # Returns additional variable specific commands
-    if variable == 'pr':
-        return '-mulc,86400'
-    return ''
 
-def create_climo_file(fp_in, fp_out, t_start, t_end, variable):
-    '''
-    Generates climatological files from an input file and a selected time range
+def create_climo_file(outdir, input_file, t_start, t_end):
+    """Generate climatological files from an input file and a selected time range.
 
-    Paramenters:
-        f_in: input file path
-        f_out: output file path
-        t_start (datetime.datetime): start date of climo period
-        t_end (datetime.datetime): end date of climo period
-        variable (str): name of the variable which is being processed
+    Parameters:
+        outdir (str): path to base directory in which to store output climo file
+        input_file (nchelpers.CFDataset): the input data file
+        t_start (datetime.datetime): start date of climo period to process
+        t_end (datetime.datetime): end date of climo period to process
 
-    Requested date range MUST exist in the input file
+    Requested date range MUST exist in the input file.
 
-    '''
+    """
+    # TODO: Are these all really legitimate variables for processing, i.e., for forming temporal means over?
```

Comment:
Trevor says it is legit to process monthly climdex variables into climatological means. However, some of these variables will already be in the form of climatological means, so we need to be sure we are not trying to reprocess these.